### ISSUE TITLE
Realtime site needs to advise user to allow popups

### ISSUE THREAD

**Issue by timlinux** [inasafe/inasafe#2937]
### Problem

On some browsers e.g. chrome, downloading the realtime reports does not work because the browser blocks popups.

<img width="1202" alt="screen shot 2016-06-09 at 10 57 16" src="https://cloud.githubusercontent.com/assets/178003/15924199/2e786f66-2e31-11e6-89e1-b26607d662fe.png">

I think it would be good (I know this is ironic) to see if the user has popups blocked and make a modal popup informing them that they should unblock popups. Alternatively, we could use an approach like we have on WebSafe where the result is displayed in a modal->iframe...
### Expected Behaviour

User should be made aware of the need to allow popups for the realtime site.

**Comment by timlinux** [comment]
Closing this in favour of the ticket in inasafe-realtime repo
